README.md

```
@@ -1,172 +1,287 @@
 
-on:
-  push:
-    branches:
-      - 'main'
-  pull_request:
-    branches:
-      - 'main'
-      - 'dev'
-  release:
-    types: [created]
+**Simpler Configuration**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)